src/lib/core/util.c

```diff
@@ -465,3 +465,20 @@ thread_sleep(double sec)
 	assert(rc == 0);
 	(void)rc;
 }
+
+int
+time_snprint(char *buf, int len)
```

Comment:
I vote for using convenient time format. Otherwise it'd require additional steps during decoding to convert time to human-readable format. Moreover using exactly the same time format for casual logs and flight recorder would simplify matching between particular logs. But in general ofc there's no much difference.

Reply:
Discussed f2f, dropped this patch. Let's use raw timestamp.